app/src/main/java/de/westnordost/streetcomplete/data/OsmModule.java

```diff
@@ -40,7 +43,7 @@
 {
 	public static final String
 		OSM_API_URL = "https://api.openstreetmap.org/api/0.6/",
-		OVERPASS_API_URL = "https://overpass-api.de/api/",
+		OVERPASS_API_WITH_ATTIC_DATA_URL = "https://overpass-api.de/api/",
```

Comment:
Why change the name of this constant?

Reply:
Because OverpassOldMapDataDao actually need attic data to work (for running tests).